CHANGES.rst

```diff
@@ -77,6 +191,10 @@ Changes to Supported Platforms
 New & Improved
 --------------
 
+- Several performance optimizations were made to hot code paths in the
+  framework to make Falcon 2.0 even faster than 1.4 in some cases.
+- Numerous changes were made to the docs to improve clarity and to provide
+  better recommendations on how to best use various parts of the framework.
 - Added a new ``headers`` property to the ``Response`` class.
 - Removed the ``six`` and ``python-mimeparse`` dependencies.
```

Comment:
Another accidental find on older additions: this line is duplicating *Removed ``six`` as a dependency.* (line 202) below.

Reply:
Good catch; fixed.